e2e/specs/edit-form.spec.ts

```diff
@@ -5,27 +5,28 @@ import {
   createValueReference,
   addFormResources,
   deleteForm,
-} from "../commands/formOperations";
+} from "../commands/form-operations";
 import { FormBuilderPage } from "../pages";
 import { Form } from "../../src/types";
 
 let form: Form = null;
+
 test.beforeEach(async ({ api }) => {
   form = await createForm(api);
   const valueReference = await createValueReference(api);
   await addFormResources(api, valueReference, form.uuid);
 });
 
-test("Should be able to edit a form", async ({ page }) => {
+test("should edit a form", async ({ page }) => {
   const formBuilderPage = new FormBuilderPage(page);
-
   await formBuilderPage.gotoFormBuilder();
-  // Click on the edit schema button
+  await expect(page.getByTestId(`editSchema${form.uuid}`)).toBeVisible();
```

Comment:
@denniskigen Ideally what needs to happen is to check all the table pages for the relevant form. Currently it just checks the first page